go/parquet/file/column_writer_test.go

```diff
@@ -430,7 +430,12 @@ func (p *PrimitiveWriterTestSuite) testDictionaryFallbackEncoding(version parque
 }
 
 func (p *PrimitiveWriterTestSuite) testDictionaryFallbackAndCompressedSize(version parquet.Version) {
-	p.GenerateData(SmallSize)
+	// skip boolean as dictionary encoding is not used
+	if p.Typ.Kind() == reflect.Bool {
+		return
+	}
+
+	p.GenerateData(LargeSize)
```

Comment:
why the change from SmallSize to LargeSize?

Reply:
Originally, I thought it was needed to ensure that a data page was written but, the smallsize works. Updated.